libs/wire-api/src/Wire/API/MLS/Message.hs

```diff
@@ -361,3 +363,36 @@ mkRemoveProposalMessage priv pub gid epoch ref = maybeCryptoError $ do
             }
   let sig = BA.convert $ sign priv pub (rmRaw tbs)
   pure (Message tbs (MessageExtraFields sig Nothing Nothing))
+
+mkAppAckProposalMessage ::
```

Comment:
This isn't used, is it?

Reply:
True. I removed it in [39d7c31](https://github.com/wireapp/wire-server/pull/2628/commits/39d7c3171b399a4f6aa0752429035ba85d6f0aa9).